lib/filter.g

```diff
@@ -344,4 +345,70 @@ BIND_GLOBAL( "CANONICAL_BASIS_FLAGS", QUO_INT(SUM_FLAGS,5) );
 
 #############################################################################
 ##
+#V  IdOfFilterByName
+#V  FilterByName
+##
+##  <#GAPDoc Label="FilterByName">
+##  <ManSection>
+##  <Func Name="IdOfFilterByName" Arg="name"/>
+##  <Func Name="FilterByName" Arg="name"/>
+##
+##  <Description>
+##  finds the id of the filter, or the filter with name <A>name</A> respectively,
+##  in the global FILTERS list. 
```

Comment:
Perhaps another reason would be that a filter can be named `Tester(FOO)`, which is not a valid gvar identifier. Of course we now changed that particular example, but the general problem remains (and is solved by `FilterByName`, I guess)